lib/rubocop/cop/naming/method_name.rb

```diff
@@ -25,14 +34,25 @@ class MethodName < Cop
         MSG = 'Use %<style>s for method names.'
 
         def on_def(node)
-          return if node.operator_method?
+          return if node.operator_method? ||
+                    allowed_method_name?(node.method_name)
 
           check_name(node, node.method_name, node.loc.name)
         end
         alias on_defs on_def
 
         private
 
+        def allowed_method_name?(method_name)
```

Comment:
Have you considered reusing [IgnoredMethodPatterns](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/mixin/ignored_method_patterns.rb)?

Reply:
Ah, that makes sense and DRY. Thank you for the suggestion!